src/app/app.component.ts

```diff
@@ -15,15 +14,16 @@ import { AppConfigService } from "./services/app-config/app-config.service";
   selector: "baw-root",
   templateUrl: "./app.component.html",
   styleUrls: ["./app.component.scss"],
+  // Need this so that router-outlet components can be styled
+  // eslint-disable-next-line @angular-eslint/use-component-view-encapsulation
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Not sure if this is need anymore. Reverted the changes which affected the sizing of page components